### PR TITLE
[hi] Localize containerization.md

### PR DESCRIPTION
**Pull request by verma-kunal**
Signed-off-by: Kunal Verma <vkunal321@gmail.com>

Localizes `content/hi/containerization.md` to Hindi

Tracking issue: #270

### CODE REVIEW

File: content/hi/containerization.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: कंटेनरीकरण
+status: Completed
+category: प्रौद्योगिकी
+---
+
+## यह क्या है 
+
+कंटेनरीकरण एक एप्लिकेशन और उसकी निर्भरता को एक [कंटेनर इमेज](/container-image/) में बंडल करने की प्रक्रिया है। कंटेनर निर्माण प्रक्रिया के लिए [ओपन कंटेनर इनिशिएटिव](https://opencontainers.org) (OCI) मानक का पालन करना आवश्यक है। जब तक आउटपुट एक कंटेनर इमेज है जो इस मानक का पालन करती है, तब तक कोई फर्क नहीं पड़ता कि किस कंटेनरीकरण उपकरण का उपयोग किया जाता है।
```
Comment (anubha-v-ardhan) [inline content/hi/containerization.md:9]:
```suggestion
कंटेनरीकरण एक एप्लीकेशन और उसकी निर्भरता को एक [कंटेनर इमेज](/container-image/) में बांधने की प्रक्रिया है। कंटेनर निर्माण प्रक्रिया के लिए [ओपन कंटेनर इनिशिएटिव](https://opencontainers.org) (OCI) मानक का पालन करना आवश्यक है। जब तक आउटपुट एक कंटेनर इमेज है जो इस मानक का पालन करती है, तब तक कोई फर्क नहीं पड़ता कि किस कंटेनरीकरण उपकरण का उपयोग किया जाता है।
```

File: content/hi/containerization.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+## समस्या  
+
+कंटेनरों के प्रचलित होने से पहले, संगठन एक ही [बेयर-मेटल मशीन](/bare_metal_machine/) पर कई अनुप्रयोगों को व्यवस्थित करने के लिए वर्चुअल मशीन (VMs) पर निर्भर थे। VMs कंटेनरों से काफी बड़े होते हैं और उन्हें चलाने के लिए एक हाइपरवाइजर की आवश्यकता होती है। इन बड़े VM टेम्प्लेट के स्टोरेज, बैकअप और ट्रांसफर के कारण, VM टेम्पलेट्स बनाना एक धीमी प्रक्रिया है। इसके अतिरिक्त, VMs कॉन्फ़िगरेशन बहाव से पीड़ित हो सकते हैं जो [अपरिवर्तनीयता](/immutable_infrastructure/) के सिद्धांत का उल्लंघन करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/containerization.md:13]:
```suggestion
कंटेनरों के प्रचलित होने से पहले, संगठन एक ही [बेयर-मेटल मशीन](/bare_metal_machine/) पर कई एप्लीकेशन को व्यवस्थित करने के लिए वर्चुअल मशीनों (VMs) पर निर्भर थे। VMs कंटेनरों से काफी बड़े होते हैं और उन्हें चलाने के लिए एक हाइपरवाइजर की आवश्यकता होती है। इन बड़े VM टेम्प्लेट के स्टोरेज, बैकअप और ट्रांसफर के कारण, VM टेम्पलेट्स बनाना एक धीमी प्रक्रिया है। इसके अतिरिक्त, VMs कॉन्फ़िगरेशन ड्रिफ्ट से पीड़ित हो सकते हैं जो [अपरिवर्तनीयता](/immutable_infrastructure/) के सिद्धांत का उल्लंघन करता है।
```

Comment (anubha-v-ardhan) [reply content/hi/containerization.md:13]: I think we should keep "configuration drift" as "कॉन्फ़िगरेशन ड्रिफ्ट" as its a commonly user phrase in industry referring to drastic changes on containers over time due to manual configuration of each container.

File: content/hi/containerization.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## समाधान 
+
+कंटेनर इमेजेज हल्की होती हैं (पारंपरिक VM के विपरीत) और कंटेनरीकरण प्रक्रिया के लिए निर्भरता की सूची वाली फ़ाइल की आवश्यकता होती है। इस फ़ाइल को संस्करण नियंत्रित किया जा सकता है और निर्माण प्रक्रिया स्वचालित हो सकती है, जिससे संगठन को अन्य प्राथमिकताओं पर ध्यान केंद्रित करने की अनुमति मिलती है जबकि स्वचालित प्रक्रियाएं निर्माण का ख्याल रखती हैं। एक कंटेनर इमेज एक अद्वितीय पहचानकर्ता द्वारा संग्रहीत की जाती है जो इसकी सटीक सामग्री और कॉन्फ़िगरेशन से जुड़ी होती है। चूंकि कंटेनरों को शेड्यूल और पुनर्निर्धारित किया जाता है, वे हमेशा अपनी प्रारंभिक स्थिति में रीसेट हो जाते हैं जो कॉन्फ़िगरेशन बहाव को समाप्त करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/containerization.md:17]:
```suggestion
पारंपरिक VM के विपरीत, कंटेनर इमेज हल्की होती हैं और कंटेनरीकरण प्रक्रिया के लिए निर्भरता की सूची वाली फ़ाइल की आवश्यकता होती है। इस फ़ाइल का संस्करण नियंत्रित किया जा सकता है और निर्माण प्रक्रिया स्वचालित हो सकती है, जिससे संगठन को अन्य प्राथमिकताओं पर ध्यान केंद्रित करने की अनुमति मिलती है जबकि स्वचालित प्रक्रियाएं निर्माण का ख्याल रखती हैं। एक कंटेनर इमेज एक अद्वितीय पहचानकर्ता द्वारा संग्रहीत की जाती है जो इसकी सटीक सामग्री और कॉन्फ़िगरेशन से जुड़ी होती है। चूंकि कंटेनरों को शेड्यूल और पुनर्निर्धारित किया जाता है, वे हमेशा अपनी प्रारंभिक स्थिति में रीसेट हो जाते हैं जो कॉन्फ़िगरेशन ड्रिफ्ट को समाप्त करता है।
```

Comment (anubha-v-ardhan) [reply content/hi/containerization.md:17]: > पारंपरिक VM के विपरीत, कंटेनर इमेज हल्की होती हैं और....

This seems a better framing of this sentence (but we lose the brackets)

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Review by jayesh-srivastava — APPROVED** [review]

File: content/hi/containerization.md (hunk 1 above)
Comment (jayesh-srivastava) [reply content/hi/containerization.md:9]: @anubha-v-ardhan I guess `बंडल करने` is right here as it is a defined technical term. We can use `बांधने` in parenthesis.

File: content/hi/containerization.md (hunk 3 above)
Comment (jayesh-srivastava) [reply content/hi/containerization.md:17]: I do think @anubha-v-ardhan 's one is a better framing.

File: content/hi/containerization.md (hunk 2 above)
Comment (jayesh-srivastava) [reply content/hi/containerization.md:13]: +1 to @anubha-v-ardhan 's one.

File: content/hi/containerization.md (hunk 1 above)
Comment (verma-kunal) [reply content/hi/containerization.md:9]: I think `बांधने` makes more sense here! Lets go with this @jayesh-srivastava

File: content/hi/containerization.md (hunk 2 above)
Comment (verma-kunal) [reply content/hi/containerization.md:13]: Oh, guess i missed `अनुप्रयोगों` here! 
Thanks for pointing out

File: content/hi/containerization.md (hunk 3 above)
Comment (verma-kunal) [reply content/hi/containerization.md:17]: Sure, lgtm

**Review by anubha-v-ardhan — APPROVED** [review]
LGTM Thankyou @verma-kunal :)

**Pull request merged**